vm/systemSmartContracts/delegation.go

```diff
@@ -1221,8 +1224,16 @@ func (d *delegation) unDelegate(args *vmcommon.ContractCallInput) vmcommon.Retur
 		return vmcommon.UserError
 	}
 
+	delegationManagement, err := d.getDelegationManagement()
+	if err != nil {
+		d.eei.AddReturnMessage("error getting minimum delegation amount " + err.Error())
+		return vmcommon.UserError
+	}
+
+	minDelegationAmount := delegationManagement.MinDelegationAmount
```

Comment:
No need for this variable

Reply:
added Jira task EN-9067 as to not forget about them